kale/embed/mpca.py

```diff
@@ -93,38 +129,57 @@ def fit(self, X):
         return self
 
     def transform(self, X):
```

Comment:
Do we have to use all bases? Can I input a lower dimension such as 2, 5, 10?

Reply:
1. Do you mean we can have mode-wise dimension reduction here?

2. The input data shape (P_1, ..., P_N) must be consistent with the training data. I will improve the documentation later.

Reply:
Not necessarily mode-wise. When we do MPCA, e.g. we have an **output** size of 20x20x20. When I use it, I may want a 10x10x10, or just a vector of 30. The current code cannot do that. And without ordidx in the matlab code, it is difficult to get a vector feature out. We can talk if you are still not clear. 

The input data size should not be P_1, .., P_N, it should be I_1, ..., I_N, right?

Reply:
The question above is not implemented yet. What if I want to get a lower dimension of 19? Let's talk in our meeting if you are not clear.